autocomplete/guide-text.md

```diff
@@ -0,0 +1,152 @@
+# Introduction
+
+
+This tutorial will walk through how to display results “as you type” in an autocomplete menu using Algolia’s search engine. We’ll be building this feature with our [autocomplete.js](https://github.com/algolia/autocomplete.js) JavaScript library.
+
+
+Let’s begin by examining a possible use case: searching for players on a team. An autocomplete dropdown will help users easily find the player they are looking for within a few keystrokes.
+
+
+#Importing Your Data
+
+
+Before you begin, you will have to import your data to our servers either via the dashboard or the API. You can find more extensive information on importing and synchronizing your data in our [import guide](https://www.algolia.com/doc/guides/indexing/import-synchronize-data).
+
+For this example, we will import player data [(download JSON file)](https://www.algolia.com/players.json) that includes the player’s name, team, and points. You can find a sample extract below.
+
+-- JSON extract here --
+
+#Settings
+
+
+## Searchable attributes
+
+
+By default, all the attributes in your records will be searchable. For this example, we want users to be able to search by player name and team, but not by points. To accomplish this functionality, we will need to set up a list of attributes to index. You can find further information on creating your searchable attributes list in our [attributes to index guide](https://www.algolia.com/doc/guides/relevance/ranking#attributes-to-index).
+
+-- Searchable attributes dashboard screenshot here --
+
+Since searching for a player by name is our primary use case, we should indicate this attribute is more important by placing it at the top of the list.
+
+
+## Custom Ranking
+
+
+To ensure users see the most relevant players, we will take into account the amount of points each player has scored (the higher the scores, the better). For that, we’ll configure the `points` attribute in the setting `customRanking`. You can refer to our [custom ranking guide](https://www.algolia.com/doc/guides/relevance/ranking#custom-ranking) for additional information regarding custom ranking setup.
+
+-- Custom ranking dashboard screenshot here --
+
+Since we would like higher scoring players to appear first, we have set the points attribute to descending sort order.
```

Comment:
For this and the above, I think we should link to the section in the dashboard or giving more information here. There's no harm in being less DRY in tutorials. What do you think?